pkg/core/stateroot/module.go

```diff
@@ -152,6 +152,10 @@ func (s *Module) CleanStorage() error {
 		b.Delete(k)
 		return true
 	})
+	s.Store.Seek(storage.SeekRange{Prefix: []byte{byte(storage.DataMPTAux)}}, func(k, _ []byte) bool {
```

Comment:
Do we remove untraceable MPT stateroots?

Reply:
We don't, I've just reproduced the old behavior here. Maybe we can keep `MPTAux` and simplify reinstantiation below?

Reply:
Fixed, I think we can just keep Aux, it has a root for block zero at this stage and that's exactly what we want to keep.

Reply:
You're right, no need to remove it.

But the question is still relevant, currently, when `RemoveUntraceableBlocks` is on, we don't perform outdated stateroots removal, so they're stored forever. Maybe we can create a separate issue for that?

Reply:
That's intentional, I don't think it should be fixed. They're treated like headers that we keep forever.